File: AppIconChanger/app/src/main/java/com/passionvirus/appiconchanger/MainActivity.java
package com.passionvirus.appiconchanger;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        AppIconChanger appIconChanger = new AppIconChanger();
        appIconChanger.setActivity(this);
        appIconChanger.addAlias("cloudy");
        appIconChanger.addAlias("drizzle");
        appIconChanger.setAlias("drizzle");
    }


}

File: Ksp/app/src/main/java/com/passionvirus/ksp/MainActivity.kt
package com.passionvirus.ksp

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.analytics.FirebaseAnalytics


class MainActivity : AppCompatActivity() {

    private lateinit var firebaseAnalytics: FirebaseAnalytics

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        KSPSampleLogger.sendEvent(
            20,
            "asfasf"
        )

    }


    fun test() {

        /*
        firebaseAnalytics.logEvent("share_image") {
            param("image_name", "asdasdf")
            param("full_text", "asda")
        }

        val bundle = Bundle()
        bundle.apply {
            putInt("aaa", 1)
        }
        firebaseAnalytics.logEvent("name", bundle)
        */

    }
}